tests/integration_tests/clouds.py

```diff
@@ -291,12 +292,15 @@ def _mount_source(instance: LXDInstance):
             subp(command.split())
 
     def _perform_launch(self, launch_kwargs, **kwargs):
-        launch_kwargs["inst_type"] = launch_kwargs.pop("instance_type", None)
-        wait = launch_kwargs.pop("wait", True)
-        release = launch_kwargs.pop("image_id")
+        instance_kwargs = deepcopy(launch_kwargs)
```

Comment:
How did this mutable being modified problem manifest as a failure in our integration tests?  Did the popped 
"wait" end up not being honored or is this now a vestigial change because you added the _wait_for_cloud function in the integration test?

Reply:
pycloudlib takes the `wait`, but cloud-init also uses the `wait` parameter, which won't get respected if we pop it off the dict. https://github.com/canonical/cloud-init/blob/main/tests/integration_tests/clouds.py#L165

It might be vestigial at this point though given that nothing in that cloud-init code should cause any kind of failure. I added it because I was really confused why my wait kwarg appeared to be disappearing.